atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -350,8 +356,11 @@ private boolean isHostHealthy(InetSocketAddress host) {
             testingContainer.runWithPooledResource(validatePartitioner);
             return true;
         } catch (Exception e) {
-            log.error("We tried to add {} back into the pool, but got an exception"
-                    + " that caused us to distrust this host further.", host, e);
+            log.warn("We tried to add {} back into the pool, but got an exception"
+                            + " that caused us to distrust this host further. Exception message was: {} : {}",
+                    SafeArg.of("host", host),
+                    SafeArg.of("exceptionClass", e.getClass().getTypeName()),
```

Comment:
nit: `getTypeName` is a new one, should be fine though `e.getClass().getName()` or `e.getClass().getCanonicalName()` might be simpler